modeldb/modeldb-run.yaml

```diff
@@ -1086,6 +1102,14 @@
 223962:
     skip: true
     comment: takes too long, need to see how to reduce time
+229276:
+    model_dir: mechanisms
+    script:
+    - cp template.hoc template.hoc.iconv.bak
+    - cp createsimulation.hoc createsimulation.hoc.iconv.bak
+    - iconv -f LATIN1 -t UTF-8 template.hoc.iconv.bak > template.hoc
+    - iconv -f LATIN1 -t UTF-8 createsimulation.hoc.iconv.bak > createsimulation.hoc
```

Comment:
just curious - do you think we should ask Rui to backport such changes into respective models?